pkg/server/sandbox_list_test.go

```diff
@@ -59,26 +59,77 @@ func TestToCRISandbox(t *testing.T) {
 
 func TestFilterSandboxes(t *testing.T) {
 	c := newTestCRIContainerdService()
-
-	testSandboxes := []*runtime.PodSandbox{
+	sandboxes := []struct {
```

Comment:
1. Why modify filterCRISandboxes?
    Because ListPodSandbox call not only c.sandboxStore.List()  but also c.taskService.List().
    The latter is difficult to fake.
2.Why modify testSandboxes?
    Because we need to query sandboxStore in filterCRISandboxes. So i must add sandboxStore before.

Reply:
SGTM